homework/index.js

```diff
@@ -30,17 +32,74 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, repositories) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-        return;
-      }
-      createAndAppend('pre', root, { text: JSON.stringify(repositories, null, 2) });
-    });
+  function getContributorInformation(contribsUrl, contribs) {
+    fetch(contribsUrl)
+      .then(data => data.json())
+      .then((jsonData) => {
+        jsonData.forEach(contributor => {
+          createAndAppend('div', contribs, { text: contributor.login, class: 'contributor' })
+          createAndAppend('img', contribs, { src: contributor.avatar_url, height: 150, widtth: 150, id: 'img' })
+          createAndAppend('div', contribs, { text: contributor.contributions })
+        })
+      })
+      .catch((err) => {
+        const root = document.getElementById('root');
+        createAndAppend('div', contribs, { text: err.message, class: 'alert-error' })
+      })
+
   }
+  function compare(a, b) {
+    if (a.name < b.name) {
+      return -1;
+    }
+    if (a.name > b.name) {
+      return 1
+    }
+    return 0;
+  }
+
+  function main(url) {
+    fetch(url)
+      .then(data => data.json())
+      .then(json => {
+        const root = document.getElementById('root');
+        const select = createAndAppend('select', root, { class: 'select' });
+        createAndAppend('option', select, { text: 'Choose your favorite repo' });
+        let sorted = json.sort((a, b) => (a.name.toLowerCase() > b.name.toLowerCase()));
+        sorted.forEach(repo => {
+          const name = repo.name;
+          createAndAppend('option', select, { text: name });
+        })
+        const wraper = createAndAppend('div', root, { class: 'wraper' });
+        const repoInfo = createAndAppend('div', wraper, { class: 'repoinfo' });
+        const contribs = createAndAppend('div', wraper, { class: 'contribs' });
+        select.addEventListener('change', evt => {
```

Comment:
I noticed that you don't load the first repo when the page loads, can you have a look at that?